sdk/model/extended_task_def.go

```diff
@@ -0,0 +1,39 @@
+/*
+ * Orkes Conductor API Server
+ *
+ * Orkes Conductor API Server
+ *
+ * API version: v2
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package model
+
+type ExtendedTaskDef struct {
```

Comment:
same comment as extended workflow def.

Reply:
Maybe, I can create a wrapper for Tags and pass it along as an argument in `RegisterTaskDefWithTags` `UpdateTaskDefWithTags` and from inside the functions create ExtendedTaskDef from TaskDef and this new Tag wrapper.

Reply:
Made the change mentioned above.